publish/publish.proj

```diff
@@ -209,6 +209,26 @@
           DependsOnTargets="GetPackagesToSign;SignFiles">
   </Target>
 
+  <Target Name="PushSignedCoreHostPackagesToAzure">
+    <Message Text="Re-uploading signed CoreHost packages to Azure" />
+
+    <ItemGroup>
+      <ItemsToPublish Include="@(_DownloadedStandardPackages)" />
```

Comment:
I think we should probably use a different item name than `@(ItemsToPublish)`.  It looks like there are other targets in this file that use that Item name as well, and we could be publishing more than we intend.

So we should either:
1. clear the Item out, and then start adding to it (not my recommendation)
2. Use a unique Item name (my recommendation)

Reply:
Good call, I've just pushed an update to name the Item `SignedPackagesToPublish`